src/makeRobot.js

```diff
@@ -38,7 +38,66 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    'name': name,
+    'wheels': wheels,
+    'version': version,
+    'coords': {
```

Comment:
```suggestion
    coords: {
```
Не обов'язково писати в лапках. Ключі і так можуть бути лише типу string